### DISCUSSION TITLE
Content suggestion: Mark MV2/MV3 API with an icon

### DISCUSSION THREAD

**Discussion by erosman** [mdn/mdn-community#189] [Content]
It would be useful to have an icon or a marker on top of the page for the APIs that are only available in MV3 (and also MV2 only).

A small visual indicator would make it easier to differentiate, without having to scroll down and read the [Browser compatibility](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting#browser_compatibility).

[tabs.executeScript()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/executeScript) **[MV2]**
[scripting](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting) **[MV3]**

**Comment by hamishwillee** [comment]
@schalkneethling Something to discuss ...

**Comment by willdurand** [comment]
> [tabs.executeScript()](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/executeScript) **[MV2]**
> [scripting](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/scripting) **[MV3]**

FWIW, scripting is also available in MV2 for Firefox
